ClassFieldTheory.lean

```diff
@@ -1,20 +1,20 @@
 import ClassFieldTheory.GroupCohomology.Examples.Examples
 import ClassFieldTheory.GroupCohomology.HerbrandInt
-import ClassFieldTheory.GroupCohomology._01_Basic
-import ClassFieldTheory.GroupCohomology._02_restriction
-import ClassFieldTheory.GroupCohomology._03_inflation
-import ClassFieldTheory.GroupCohomology._04_TateCohomology_def
-import ClassFieldTheory.GroupCohomology._05_TrivialCohomology
-import ClassFieldTheory.GroupCohomology._06_LeftRegular
-import ClassFieldTheory.GroupCohomology._07_coind1_and_ind1
-import ClassFieldTheory.GroupCohomology._08_DimensionShift
-import ClassFieldTheory.GroupCohomology._09_CyclicGroup
-import ClassFieldTheory.GroupCohomology._10_inflationRestriction
-import ClassFieldTheory.GroupCohomology._11_TrivialityCriterion
-import ClassFieldTheory.GroupCohomology._12_augmentationModule
-import ClassFieldTheory.GroupCohomology._13_HerbrandQuotient
-import ClassFieldTheory.GroupCohomology._14_SplittingModule
-import ClassFieldTheory.GroupCohomology._15_corestriction
+import ClassFieldTheory.GroupCohomology.«01_Basic»
+import ClassFieldTheory.GroupCohomology.«02_restriction»
+import ClassFieldTheory.GroupCohomology.«03_inflation»
+import ClassFieldTheory.GroupCohomology.«04_TateCohomology_def»
+import ClassFieldTheory.GroupCohomology.«05_TrivialCohomology»
+import ClassFieldTheory.GroupCohomology.«06_LeftRegular»
+import ClassFieldTheory.GroupCohomology.«07_coind1_and_ind1»
+import ClassFieldTheory.GroupCohomology.«08_DimensionShift»
+import ClassFieldTheory.GroupCohomology.«09_CyclicGroup»
+import ClassFieldTheory.GroupCohomology.«10_inflationRestriction»
+import ClassFieldTheory.GroupCohomology.«11_TrivialityCriterion»
+import ClassFieldTheory.GroupCohomology.«12_augmentationModule»
+import ClassFieldTheory.GroupCohomology.«13_HerbrandQuotient»
+import ClassFieldTheory.GroupCohomology.«14_SplittingModule»
+import ClassFieldTheory.GroupCohomology.«15_corestriction»
```

Comment:
What's going on here? I know this fixes the blueprint, but I thought the solution was to remove the numbers, not to make the file names untypable!

Reply:
FWIW I also think the numbers are a terrible idea now, they were useful earlier when Richard was working out the flow of the argument but now they're just a distraction.

Reply:
I will prepare a PR

Reply:
First PR for 01 is here: #41. Second PR will rename all other numbered files.